script/set-version

```diff
@@ -1,30 +1,30 @@
-#!/usr/bin/env bash
+#!/bin/sh -eux
 
 set -euo pipefail
 
-pvp="$1"
-semantic="$2"
-
-if [ -z "$pvp" ] || [ -z "$semantic" ]; then
+if [ $# -ne 2 ]; then
     echo "Usage: bump-version <VERSION> <DEPENDENCY>"
     exit 1
 fi
 
+pvp="$1"
+semantic="$2"
```

Comment:
The check around arguments wasn't working in my shell, I think checking the length of arguments and printing usage before failing is the more robust solution.